developer_guides/debugability/logger/index.rst

```diff
@@ -0,0 +1,106 @@
+.. _dbg-logger:
+
+Logger
+######
+
+Sof-logger is used to print logs delivered from FW dma_trace mechanism, by searching log
+entries in ldc file generated by rimage. Every entry declared in FW is placed in elf output file (e.g. sof-apl) in 
+.static_log_entries section in a form of struct defined in sof/src/include/sof/trace.h in sof fw repo.
+
+Ldc file contains snd_sof_logs_header (defined in rmbox/logger_convert.c) 
+following by .static_log_entries section incorporated from FW elf file (e.g. sof-apl).
+snd_sof_logs_header contains basic information about .static_log_entries section
+like base_address and data_length. Sof-logger works by reading entry parameters value and
+entries addresses from FW dma_trace mechanism and searching suitable entry in ldc file
+by its address.
+
+Logger usage
+************
+
+Usage sof-logger <option(s)> <file(s)>
+
+-h 					help
+-l ldc_file			Specify the ldc file
+-i in_file			Get traces from in_file, instead of the default "/sys/kernel/debug/sof/etrace"
+-o out_file 		Specify the output file, instead of default stdout
+-t					Get traces from "/sys/kernel/debug/sof/trace", instead of the default "/sys/kernel/debug/sof/etrace"
+-p					Get traces from stdin, instead of the default "/sys/kernel/debug/sof/etrace"
+-c					Set timestamp clock in MHz
+-e					Enable checking firmware version with default verification file "/sys/kernel/debug/sof/fw_version"
+-v ver_file 		Enable checking firmware version with ver_file file, instead of default: "/sys/kernel/debug/sof/fw_version"
+-s					Take a snapshot of state
+
+
+Examples:
+
+- Get traces from "/sys/kernel/debug/sof/etrace" file, verifies fw_version with
+  "/sys/kernel/debug/sof/fw_version" and prints logs to stdout 
+ 
+.. code-block:: bash
+
+	$ sof-logger -l ldc_file -e
```

Comment:
Just one thing - indentation should be 3 spaces instead of tab

Reply:
updated